Kommander workspace edits and updates especially delete workspace was… (#3563)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.4/workspaces/create-workspaces/index.md

![Workspace Form Advanced Annotations](/dkp/kommander/1.4/img/workspace-annotations.png)
<br />_Create Workspace Advanced Options_

When creating or editing a workspace, you can use the Advanced Options to add, edit, or delete annotations and labels to your workspace. Both the annotations and labels are applied to the workspace's namespace.

Labels that are added to a workspace, are also applied to all of the clusters in the workspace.

You can also create a **custom namespace** for the Workspace you are creating.
By default, a unique ID Namespace will be auto-generated based on the first 57 characters of your workspace name plus 5 unique alphanumeric characters.
![Workspace default naming](/dkp/kommander/1.4/img/workspace-default-name.png)
<br />_Workspace namespace fields_

> To override this default behavior, add what you specifically want to be for Workspace's namespace.

Reviewer: ```suggestion
To override this default behavior, add you specific Workspace namespace name.
```

Reviewer 2: ```suggestion
To override this default behavior, add your custom namespace name for the Workspace.
```
english is hard, still not very happy with this but maybe it'll get us closer to something that sounds nice lol

Author: Definitely less clunky, thanks @gracedo

![Workspace default naming](/dkp/kommander/1.4/img/workspace-custom-name.png)
<br />_Adding a custom namespace for your Workspace_

> <p class="message--warning"><strong>NOTE: </strong>By setting a custom ID for the Namespace, Kommander cannot guarantee safety and uniqueness of the namespace across all clusters, and that it may override any namespaces of the same name on imported clusters.</p>

Reviewer: ```suggestion
<p class="message--warning"><strong>NOTE: </strong>Setting a custom ID for the Namespace in Kommander  does not guarantee safety and uniqueness of the namespace across all clusters. This may override any namespaces, with the same name, on imported clusters.</p>
```

Reviewer 2: ```suggestion
<p class="message--warning"><strong>NOTE: </strong>Setting a custom ID for the namespace in Kommander does not guarantee safety and uniqueness of the namespace across all clusters. This may override any namespaces, with the same name, on attached clusters.</p>
```
(minor edits on russ's suggestion)

Author: Thanks both @gracedo @russurquhart1
